src/store/Menu.ts

```diff
@@ -13,6 +13,7 @@ export class Menu implements Module {
         exportDisabled: await ManagedStorage.get("disableExport"),
         enforcePassword: await ManagedStorage.get("enforcePassword"),
         enforceAutolock: await ManagedStorage.get("enforceAutolock"),
+        showQrDisabled: await ManagedStorage.get("disableExport"),
```

Comment:
Why did you add a line here? The exact same thing happens on line 13

Reply:
Ooooops